spec/services/offer_validations_spec.rb

```diff
@@ -138,6 +138,16 @@
         end
       end
 
+      context 'when a provider attemps to make an offer with other application as conditions not met' do
```

Comment:
```suggestion
      context 'when a provider attempts to make an offer when conditions not met sibling exists' do
```

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9141/commits/f78a6bf173dff8eb57aefcd472371c6ad5589a45